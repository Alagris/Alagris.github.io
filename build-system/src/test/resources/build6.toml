projectName = "test project 1"
[[source]]
path = "mirror.star"
[[source]]
path = "function3.star"
