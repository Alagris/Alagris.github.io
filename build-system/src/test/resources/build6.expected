''
'a'
'b'
'z'
Transducer 'auto_exponentials' not found!
Transducer 'auto_exponentials' not found!
Transducer 'auto_exponentials' not found!
Transducer 'referenced_across_files' not found!
Transducer 'multiple_types' not found!
Transducer 'used_ahead_of_definition' not found!
Transducer 'used_ahead_of_definition' not found!
Transducer 'used_ahead_of_definition' not found!
